modules/engine.py

```diff
@@ -22,6 +23,14 @@ def main_loop(self):
                 play = False
             elif command == "help":
                 self.display_help()
+            elif command == "begin":
+                game = modules.world.Room('resources\Game_Setup.csv')
```

Comment:
resources\Game_Setup.csv would be more flexible if it is written as os.path.join("resources", "Game_Setup.csv")